<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1009</width>
    <height>639</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="main_content_">
   <layout class="QVBoxLayout" name="vertical_layout_1_">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>0</number>
    </property>
    <property name="topMargin">
     <number>0</number>
    </property>
    <property name="rightMargin">
     <number>0</number>
    </property>
    <property name="bottomMargin">
     <number>0</number>
    </property>
    <item>
     <widget class="DriverOverridesNotificationBanner" name="driver_overrides_notification_banner_" native="true"/>
    </item>
    <item>
     <widget class="TabWidget" name="main_tab_widget_">
      <property name="currentIndex">
       <number>7</number>
      </property>
      <property name="movable">
       <bool>true</bool>
      </property>
      <widget class="QWidget" name="navigation_tab_">
       <attribute name="title">
        <string>NAVIGATION</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="start_tab_">
       <attribute name="title">
        <string>START</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontal_layout_" stretch="1,5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="NavigationListWidget" name="start_list_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Welcome</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Recent traces</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>About</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="start_stack_"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="overview_tab_">
       <attribute name="title">
        <string>OVERVIEW</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontal_layout_1_" stretch="1,5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="NavigationListWidget" name="overview_list_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>Summary</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>System information</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="overview_stack_"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tlas_tab_">
       <attribute name="title">
        <string>  TLAS  </string>
       </attribute>
       <layout class="QHBoxLayout" name="tlas_layout_" stretch="1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ScaledTabWidget" name="tlas_sub_tab_">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="TlasViewerPane" name="tlas_viewer_tab_">
           <attribute name="title">
            <string>Viewer</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a 3D rendering of the selected TLAS.</string>
           </attribute>
          </widget>
          <widget class="TlasInstancesPane" name="tlas_instances_tab_">
           <attribute name="title">
            <string>Instances</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a table of all instances of the selected TLAS.</string>
           </attribute>
          </widget>
          <widget class="BlasListPane" name="tlas_blas_list_tab_">
           <attribute name="title">
            <string>BLAS list</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a table of all bottom level acceleration structures in the selected TLAS.</string>
           </attribute>
          </widget>
          <widget class="TlasPropertiesPane" name="tlas_properties_tab_">
           <attribute name="title">
            <string>Properties</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a list of properties for the selected TLAS.</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="blas_tab_">
       <attribute name="title">
        <string>  BLAS  </string>
       </attribute>
       <layout class="QHBoxLayout" name="blas_layout_">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="ScaledTabWidget" name="blas_sub_tab_">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="BlasViewerPane" name="blas_viewer_tab_">
           <attribute name="title">
            <string>Viewer</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a 3D rendering of the selected BLAS.</string>
           </attribute>
          </widget>
          <widget class="BlasInstancesPane" name="blas_instances_tab_">
           <attribute name="title">
            <string>Instances</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a table of all instances of the selected BLAS.</string>
           </attribute>
          </widget>
          <widget class="BlasTrianglesPane" name="blas_triangles_tab_">
           <attribute name="title">
            <string>Triangles</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a table of all triangles in the selected BLAS.</string>
           </attribute>
          </widget>
          <widget class="BlasGeometriesPane" name="blas_geometries_tab_">
           <attribute name="title">
            <string>Geometries</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a table of all geometries in the selected BLAS.</string>
           </attribute>
          </widget>
          <widget class="BlasPropertiesPane" name="blas_properties_tab_">
           <attribute name="title">
            <string>Properties</string>
           </attribute>
           <attribute name="toolTip">
            <string>Show a list of properties for the selected BLAS.</string>
           </attribute>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="ray_tab_">
       <attribute name="title">
        <string>  RAY  </string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontal_layout_6_" stretch="1">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="ray_history_valid_switch_">
          <property name="currentIndex">
           <number>0</number>
          </property>
          <widget class="QWidget" name="table_empty_">
           <layout class="QVBoxLayout" name="table_empty_layout_">
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <spacer name="vertical_spacer_2_">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_1_">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="text">
               <string/>
              </property>
              <property name="pixmap">
               <pixmap>:/Resources/assets/stop_128x128.png</pixmap>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ScaledLabel" name="label_2_">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>The loaded trace does not contain ray history information.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="ScaledLabel" name="label_3_">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <pointsize>12</pointsize>
               </font>
              </property>
              <property name="text">
               <string>If ray history information is required, please ensure ray history is enabled in the Radeon Developer Panel when capturing ray trace information.</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignmentFlag::AlignCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="vertical_spacer_3_">
              <property name="orientation">
               <enum>Qt::Orientation::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
          <widget class="ScaledTabWidget" name="ray_sub_tab_">
           <property name="currentIndex">
            <number>0</number>
           </property>
           <widget class="RayHistoryPane" name="ray_history_tab_">
            <attribute name="title">
             <string>Dispatches</string>
            </attribute>
            <attribute name="toolTip">
             <string>Show a list of trace ray events, their timeline and a 3D rendering of the scene.</string>
            </attribute>
           </widget>
           <widget class="RayInspectorPane" name="ray_inspector_tab_">
            <attribute name="title">
             <string>Inspector</string>
            </attribute>
            <attribute name="toolTip">
             <string>Inspect a particular ray from a dispatch.</string>
            </attribute>
           </widget>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="spacer_tab_">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <attribute name="title">
        <string>SPACER</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="settings_tab_">
       <attribute name="title">
        <string>SETTINGS</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontal_layout_7_" stretch="1,5">
        <property name="spacing">
         <number>0</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="NavigationListWidget" name="settings_list_">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <item>
           <property name="text">
            <string>General</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Themes and colors</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Keyboard shortcuts</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QStackedWidget" name="settings_stack_"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="ScaledMenuBar" name="menu_bar_">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1009</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ScaledLabel</class>
   <extends>QLabel</extends>
   <header>qt_common/custom_widgets/scaled_label.h</header>
  </customwidget>
  <customwidget>
   <class>NavigationListWidget</class>
   <extends>QListWidget</extends>
   <header>qt_common/custom_widgets/navigation_list_widget.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledMenuBar</class>
   <extends>QMenuBar</extends>
   <header>qt_common/custom_widgets/scaled_menu_bar.h</header>
  </customwidget>
  <customwidget>
   <class>ScaledTabWidget</class>
   <extends>QTabWidget</extends>
   <header>qt_common/custom_widgets/scaled_tab_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabWidget</class>
   <extends>QTabWidget</extends>
   <header>qt_common/custom_widgets/tab_widget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TlasViewerPane</class>
   <extends>QWidget</extends>
   <header>views/tlas/tlas_viewer_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlasViewerPane</class>
   <extends>QWidget</extends>
   <header>views/blas/blas_viewer_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlasListPane</class>
   <extends>QWidget</extends>
   <header>views/tlas/blas_list_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TlasInstancesPane</class>
   <extends>QWidget</extends>
   <header>views/tlas/tlas_instances_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlasInstancesPane</class>
   <extends>QWidget</extends>
   <header>views/blas/blas_instances_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlasTrianglesPane</class>
   <extends>QWidget</extends>
   <header>views/blas/blas_triangles_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlasGeometriesPane</class>
   <extends>QWidget</extends>
   <header>views/blas/blas_geometries_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TlasPropertiesPane</class>
   <extends>QWidget</extends>
   <header>views/tlas/tlas_properties_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BlasPropertiesPane</class>
   <extends>QWidget</extends>
   <header>views/blas/blas_properties_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RayHistoryPane</class>
   <extends>QWidget</extends>
   <header>views/ray/ray_history_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>RayInspectorPane</class>
   <extends>QWidget</extends>
   <header>views/ray/ray_inspector_pane.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>DriverOverridesNotificationBanner</class>
   <extends>QWidget</extends>
   <header>qt_common/custom_widgets/driver_overrides_notification_banner.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>

